language/fast_text.py

```diff
@@ -0,0 +1,45 @@
+try:
```

Comment:
Hi @noviluni, 
Can I use this kind of in program installation of the additional libraries?